### ISSUE TITLE
Creating Note from Card does not implement set template?

### ISSUE THREAD

**Issue by frvkl** [mgmeyers/obsidian-kanban#127]
Obsidian version: 0.12.3
Version Kanban plugin: 0.4.0

I have set a template for note creation from card:
![image](https://user-images.githubusercontent.com/19724110/120445782-41fd7e80-c389-11eb-94e5-ba8a392f4461.png)

Unfortunately, when creating a note from a Kanban card, the template is not inserted into the newly created note.

**Comment by mgmeyers** [comment]
@frvkl Just to clarify, are you using regular obsidian templates, or templater?

**Comment by frvkl** [comment]
I am using regular Obsidian templates. Thank you.

**Comment by mgmeyers** [comment]
@frvkl Can you provide some more details about how your templates are set up, or what specifically happens when you create a card? Templates seem to work fine for me, but I may be doing something differently.


https://user-images.githubusercontent.com/2694747/121229763-3c011500-c843-11eb-90fd-06bb0d44d83b.mov

**Comment by frvkl** [comment]
@mgmeyers Thank you for checking and demonstrating. I checked in detail myself just now, and I found out this happens due to my workflow.

When I use the create note from card function, it does work and the template is applied to the new note. When I type a [[wikilink]] in the card for a non-existing note, and then click on the wikilink to create the note, no template is applied. Perhaps that is outside of the plugin scope?
